logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -145,6 +155,11 @@ class MLSConversationDataSource(
         }
 
     override suspend fun updateKeyingMaterial(groupID: GroupID): Either<CoreFailure, Unit> =
+        executeOperation(groupID) {
```

Comment:
isn't better instead of having two functions, wrap the body of the function with executeOperation wrapper? like other wrappers. 
I know that we need to keep the main functions unwrapped to use them in other places and wrap the whole new function[my assumption] but still we can improve this of having duplicated functions.